File: devpractice/machinecoding/parkingLot/untitled/src/main/java/org/example/Vehicle.java
package org.example;

public class Vehicle {
    VehicleType vehicleType;
    String vehicleNumber;
    String color;
    String ticket;

    public Vehicle(VehicleType vehicleType, String vehicleNumber, String color) {
        this.vehicleType = vehicleType;
        this.vehicleNumber = vehicleNumber;
        this.color = color;
    }
}

File: devpractice/machinecoding/ParkingLot/ParkingLot/Vehicle.cs
namespace ParkingLot
{
    public class Vehicle
    {
        public int id { get; set; }
        public VehicleType type { get; set; }
        public Ticket ticket { get; set; } = new Ticket();
        public Vehicle(int id, VehicleType type)
        {
            this.id = id;
            this.type = type;
        }
    }

}
